File: client/dotnet/src/UdpPacket.cs
using System;
using System.Net;
using System.Net.Http;
using System.Threading.Tasks;
using System.Diagnostics;
using System.Net.Sockets;
using System.Collections.Generic;
using System.Security.Cryptography;
using System.Text;

namespace FfClient
{
    internal class UdpPacket
    {
        public byte[] Packet { get; set; }

        public int Length { get; set; }
    }
}

File: client/java/src/main/java/com/timetoogo/ffclient/UdpPacket.java

package com.timetoogo.ffclient;

class UdpPacket {
    private byte[] value;
    private int length;

    public UdpPacket(byte[] value, int length) {
        this.value = value;
        this.length = length;
    }

    public int getLength() {
        return length;
    }

    public byte[] getValue() {
        return value;
    }
}
